CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 #### Argument parser
 * Simplified reading file extensions from formatted files in the input/output file validators.
+* API break: The seqan3::value_list_validator is now constructible from a range or a parameter pack but not from a
```

Comment:
Can you add an example? E.g.
Before `seqan3::value_list_validator{{1,2,3}}` after `seqan3::value_list_validator{1,2,3}` or something.